### PR TITLE
feat: some improvements

### PR DESCRIPTION
**Pull request by hoegertn**
I made some changes that were necessary on my machine to make it work on a project.

### CODE REVIEW

File: src/utils/resolveLambdaDetailsFromManifest.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ const resolveLambdaDetailFromManifest = async (
   detail: CloudFormation.StackResourceDetail;
   lambdaManifest: LambdaManifestType;
 }> => {
-  const cfn = new CloudFormation();
+  const cfn = new CloudFormation({maxRetries: 10});
```
Comment (kirkness) [inline src/utils/resolveLambdaDetailsFromManifest.ts:10]: What were the errors you were seeing in your case? Perhaps you have a lot of lambdas, so you were hitting a rate limit?

**Review by kirkness — COMMENTED** [review]
Thanks a bunch for your contribution, it all makes sense, will get this merged/released after running locally today.

File: src/utils/resolveLambdaDetailsFromManifest.ts (hunk 1 above)
Comment (hoegertn) [reply src/utils/resolveLambdaDetailsFromManifest.ts:10]: Yes, I have about 30 Lambdas and I got rate limited. So retries solve this.

Comment (kirkness) [reply src/utils/resolveLambdaDetailsFromManifest.ts:10]: Gotcha!

**Comment by kirkness** [comment]
Looks good to me! Cheers @hoegertn

**Pull request merged**